schemas/oic.wk.res-schema.json

```diff
@@ -8,24 +8,20 @@
       "properties": {
```

Comment:
You are correct.  The example in the Endpoint submission shows that; but there is no indication in the submission to change the appropriate schema files for "/oic/res", so I just overlooked it. I will work with Jin to get this resolved and the Endpoint submission cleared up.